Common/VR/PPSSPPVR.cpp

```diff
@@ -684,7 +684,7 @@ bool StartVRRender() {
 		vrCompat[VR_COMPAT_SKYPLANE] = PSP_CoreParameter().compat.vrCompat().Skyplane;
 
 		// Set customizations
-		__DisplaySetFramerate(g_Config.bForce72Hz ? 72 : 60);
+		g_Config.iDisplayRefreshRate = g_Config.bForce72Hz ? 72 : 60;
```

Comment:
Just wondering whether changing the `g_Config` var programmatically like this is a common practice?
Did it changed the value at the UI too?

Reply:
Yeah, do not write temporary state to the config value ever. Only explicitly from UI.

Reply:
@anr2me Force72Hz is true by default, and since this is a VR-only option not visible to pancake users, I'd be locked out of DisplayRefreshRate for no reason.

We can instead check we're in VR when setting the framerate, that way the config doesn't get programmatically written to.